modules/search/pages/quick-index-field-options.adoc

```diff
@@ -0,0 +1,69 @@
+= Quick Index Field Options 
+:page-topic-type: reference
+:description: When you create a Search index with the Quick Editor, you must set options for each field you add to the index. 
+
+[abstract]
+{description}
+
+For more information about how to create a Search index with the Quick Editor, see xref:create-quick-index.adoc[]. 
+
+The following options are available for fields in the Quick Editor: 
+
+|====
+|Option |Description 
+
+|Index this field as an identifier (Text Only) a|
+
+To use the field to set the document's type in the index, select *Index this field as an identifier*. 
+
+To not use the field as the document's type, clear *Index this field as an identifier*. 
+
+|Language (Text Only) a|
+
+Select the language for the content inside a text field. 
+
+The Search Service automatically applies an xref:customize-index.adoc#analyzers[analyzer] to the field's contents based on the selected language. 
+
+For more information about the available language options, see xref:quick-index-supported-languages.adoc[].
+
+|Include in search results a|
+
+To include content from the field in search results, select *Include in search results*. 
+
+To exclude the field's content from search results, clear *Include in search results*. 
```

Comment:
I'm finding these oppositional pairs of sentences hard to read, as they look almost identical.

Reply:
That's why I tried to change the order of the words in each sentence.

Reply:
OK, take it as a data-point of feedback to consider iterating on / leaving as is as you prefer 👍